corporate/views/support.py

```diff
@@ -95,6 +95,21 @@ class SupportRequestForm(forms.Form):
     request_message = forms.CharField(widget=forms.Textarea)
 
 
+class DemoRequestForm(forms.Form):
+    MAX_INPUT_LENGTH = 50
+    SORTED_ORG_TYPE_NAMES = sorted(
+        ([org_type["name"] for org_type in Realm.ORG_TYPES.values() if not org_type["hidden"]]),
+    )
+    full_name = forms.CharField(max_length=MAX_INPUT_LENGTH)
+    email = forms.EmailField()
+    role = forms.CharField(max_length=MAX_INPUT_LENGTH)
+    organization_name = forms.CharField(max_length=MAX_INPUT_LENGTH)
+    organization_type = forms.CharField()
+    organization_website = forms.URLField(required=True)
+    expected_user_count = forms.CharField(max_length=MAX_INPUT_LENGTH)
+    message = forms.CharField(widget=forms.Textarea)
```

Comment:
Probably it'd be a good refactor to split the internal support agent pages from the externally-facing support pages, but that can be a follow-up that we discuss in Zulip; I'm not sure which of the two we should rename :)